lib/modules/apostrophe-ui/public/js/ui.js

```diff
@@ -18,13 +18,71 @@ apos.define('apostrophe-ui', {
       self.enableActionable();
       self.enhanceDropdowns();
       self.enableAjax();
+      self.enableTooltips();
     });
   },
 
   construct: function(self, options) {
 
     self.options = options;
 
+    // Utility that lets you add a tooltip to any HTML element using
+    // the data-apos-tooltip="YR TOOLTIP MSG" pattern
+
+    self.enableTooltips = function() {
+
+      var $tooltip = $('[data-apos-tooltip-template]');
+      var $clone = $tooltip.clone();
+      var $body = $('body');
+      $clone
+        .removeAttr('data-apos-tooltip-template')
+        .attr('data-apos-tooltip', 'true')
+        .removeClass('apos-hidden');
+
+      var removeTooltip = function() {
+        var $this = $(this);
+        var $tooltip = $body.find('[data-apos-tooltip]:first');
+        $tooltip.one('webkitTransitionEnd otransitionend oTransitionEnd msTransitionEnd transitionend', function() {
+          $tooltip.remove();
+        })
+        $tooltip.css('opacity', 0);
+      }
+
+       $('[data-apos-tooltip]').on({
```

Comment:
Can't seem to delegate mouseenter/mouseover events from body, they don't fire, example code doesn't work. After a quick look, we don't seem to use that pattern in the core for hover events.

Let's link up to figure out where to put this